packages/ckeditor5-list/src/todolist/todolistediting.ts

```diff
@@ -392,6 +398,29 @@ export default class TodoListEditing extends Plugin {
 	}
 }
 
+/**
+ * Returns an upcast converter for to-do list items.
+ */
+function todoListItemUpcastConverter(): GetCallback<UpcastElementEvent> {
+	return ( evt, data, conversionApi ) => {
+		const { writer } = conversionApi;
+
+		if ( data.modelRange ) {
```

Comment:
Please return from the converter early so we do not keep the whole body in a single `if` statement.